### PR TITLE
Fix #118061

### LINKED ISSUE #118061: bagi.co.in
### Issue URL (Anti Adblock Script)
 [https://bagi.co.in/bitcoin/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fbagi.co.in%2Fbitcoin%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.10 nightly 4
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (180), <br>Disable cache for third-party requests, <br>Block WebRTC, <br>Block Push API, <br>Block Location API, <br>Block Flash
DNS filtering: | server: `https://dns.adguard.com/dns-query`<br>filters: `User rules`
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>EasyPrivacy, <br/>Legitimate URL Shortener, <br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Turkish, <br/>AdGuard Russian
Userscripts: | https://userscripts.adtidy.org/beta/assistant/4.3/assistant.user.js,<br>https://userscripts.adtidy.org/beta/adguard-extra/1.0/adguard-extra.user.js,<br>https://userscripts.adtidy.org/beta/popup-blocker/2.5/popupblocker.user.js

### PR DESCRIPTION
**Pull request by gork7777**
##### To avoid invalid pull requests, please check and confirm following checkboxes:

        
  - [x] This is not an ad/bug report;
  - [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
  - [x] I have performed a self-review of my own changes;
  - [x] My changes do not break web sites, apps and files structure.

##### If the problem does not fall under any category that is listed here, please contact our tech support: `support@adguard.com`

  - [ ] Missed ads or ad leftovers;
  - [ ] Website or app doesn't work properly;
  - [x] AdGuard gets detected on a website;
  - [ ] Missed analytics or tracker;
  - [ ] Social media buttons — share, like, tweet, etc;
  - [ ] Annoyances — pop-ups, cookie warnings, etc.
  
### What issue is being fixed?
https://github.com/AdguardTeam/AdguardFilters/issues/118061

### Add your comment and screenshots

**2.** Screenshots

<details><summary>Screenshot 1:</summary>

image

</details>

<details><summary>Screenshot N:</summary>

image

</details>

### Terms

  - [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met.

### CODE REVIEW

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -681,6 +681,8 @@ smallseotools.com#@#.adsbygoogle
 !
 !
 ! anti-adb
+! https://github.com/AdguardTeam/AdguardFilters/issues/118061
+||bagi.co.in/*/js/check.js
```
Comment (Alex-302) [inline EnglishFilter/sections/antiadblock.txt:685]: Wrong place - between existing comment+rule.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/antiadblock.txt (hunk 1 above)
Comment (Alex-302) [reply EnglishFilter/sections/antiadblock.txt:685]: @gork7777

Comment (gork7777) [reply EnglishFilter/sections/antiadblock.txt:685]: @Alex-302 Which row should it be in?

Comment (Alex-302) [inline EnglishFilter/sections/antiadblock.txt:685]:
```suggestion
! https://github.com/AdguardTeam/AdguardFilters/issues/118061
||bagi.co.in/*/js/check.js
! anti-adb
```

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**